app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+
+import requests
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    """Get weather data from an external API and print the result."""
+    api_key = os.environ.get("WEATHER_API_KEY")
+    url = ("http://api.weatherapi.com/v1/current.json?"
+           f"key={api_key}&q=Paris&lang=en")
+
+    response = requests.get(url)
+
+    if response.status_code == 200:
+        data = response.json()
+
+        location = data["location"]["name"]
+        temperature = data["current"]["temp_c"]
+        condition = data["current"]["condition"]["text"]
+        print(f"The weather in {location} is {temperature}°C and {condition}")
+    if response.status_code != 200:
+        print(f"Error: {response.status_code} - {response.reason}")
+    else:
+        print("Error: Could not retrieve weather data")
```

Comment:
why if status_code = 200 u print "Error: Could not retrieve weather data"
200 = OK
remove redundant part